application/console/migrations/m181025_155801_create_method_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+use yii\db\Migration;
+
+/**
+ * Handles the creation of table `method`.
+ */
+class m181025_155801_create_method_table extends Migration
+{
+    /**
+     * @inheritdoc
+     */
+    public function up()
```

Comment:
I don't know if it will actually make a difference in this case, but I think our general practice is to use `safeUp()`/`safeDown()` rather than `up()`/`down()` in case there is some benefit to doing the changes as one transaction.